patches/chrome-browser-page_load_metrics-page_load_metrics_initialize.cc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc b/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc
+index 9bb1490b5d45eb7ed208f1f6b0332a1e3732aa29..5667b397f38abae2d3f4a54a004769746c644ac1 100644
+--- a/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc
++++ b/chrome/browser/page_load_metrics/page_load_metrics_initialize.cc
+@@ -62,6 +62,8 @@
+ #include "extensions/common/constants.h"
+ #endif
+ 
++#include "brave/browser/brave_performance_observers.h"
```

Comment:
just for future reference, we always want to keep patches as small as possible so we never add whitespace and also any required header files should be added through chromium_src overrides, but in this case I don't think we want this patch at all